graphql/resolvers.go

```diff
@@ -2,11 +2,14 @@ package graphql
 
 import (
 	"context"
+	"sort"
+	s "strings"
```

Comment:
please do not alias imports. it decreases readability and the ability to search the code base.